src/infra/main/template.yaml

```diff
@@ -380,6 +403,14 @@ Resources:
     Properties:
       FunctionName: !Sub '${AWS::StackName}-AccountDeletionProcessorFunction'
       Description: "Updates an account intervention status indicating that the account has been deleted."
+      DeploymentPreference:
+        Type: !If
```

Comment:
just to triple check / add a reminder here, this condition will be inverted when merging this in. This is done like this so that we can test in dev first

Reply:
Yep so currently set to this so that we can test in Dev, but once ready to be merged in will change over from  `Isnternal` to `IsProd`